src/compas/scene/scene.py

```diff
@@ -52,17 +52,8 @@ def clear(self):
     def redraw(self):
         drawn_objects = []
         for sceneobject in self.sceneobjects:
-            drawn_object = sceneobject.draw()
-
-            # TODO: unify output of draw(), so we don't have to do this
-            if isinstance(drawn_object, (list, tuple)):
-                for item in drawn_object:
-                    if isinstance(item, (list, tuple)):
-                        drawn_objects.extend(item)
-                    else:
-                        drawn_objects.append(item)
-            else:
-                drawn_objects.append(drawn_object)
+            sceneobject.clear()
```

Comment:
would it not be better to collect the objects that have to be cleared and do this all at once? at least in rhino it is a costly operation...